### ISSUE TITLE
Can't get the latest homebridge on vm hyper-v according to guide with node.js update necessary and not working as instructed and more

### ISSUE THREAD

**Issue by candybars1** [homebridge/homebridge#3605]
Edit: Everything below will be fixed if you just update the guide to the latest image homebridge/homebridge (new node.js, ubuntu 22.04 based, etc) would save terrible headaches

1. Installed Homebridge on Hyper-v windows according to this guide. After not pulling due to delay in resolving registry and realizing host dns needed to use public DNS to work (should be corrected in guide), I ran into a message duing the install that node.js isn't up to date, followed by not being able to update to latest UI with message:

>  Homebridge v1.8.1 requires Node.js v18.15.0 or later. You are currently running Node.js v18.13.0 (homebridge UI)

2. THe guide at the bottom indicates the update inside the VM, attempted when container was running/stopped in both cases it did not work (https://github.com/homebridge/homebridge/wiki/Install-Homebridge-on-Windows-10-Using-Hyper-V):
> To update Node.js to the latest LTS version of Node.js run the following command in the Homebridge terminal:
> 
> sudo hb-service update-node

cudo hb-service update-node"
ommand was not accepted (hb-service unknow command) when container was running or not, despite the very clear note in the guide, and the fact the vm link requires that update.  

3. The exact VM OS and version is a stripped down version of linux I could not get any info on elsewhere and is

>  Boot2Dockerversion 19.03.16 (TCL 11) 
and again does not suppor hb-service command. 

4. Result: despite intense research i am unable to find reference to the correct command to update node.js. 

5. Also in the link to the annoucement on homebridge ui (all git pages for homerige) it warns that : "Your version of Linux does not meet the GLIBC version requirements to use this tool to upgrade Node.js.

> To check your GLIBC version you can run the command getconf 
> GNU_LIBC_VERSION on Linux and Raspbian platforms.
> 
> To Install Node.js:	Needed GLIBC Version
> 18	>=2.28
> 20	>=2.31"

Unfamiliar with what this means, and unable to find out how to check (I can't even copy paste from VM connecting with remote desktop), I am left stranded. 

To summarize given the above, please explain 1. how to update node.js with command that works 2, to safe version (inc GLIBC?) and 3.  update:1.the guide for homebridge hyper v to reflect this 4. proabbly update the VM install to include the necessary node.js. 

Please it into account it took me a week to get thus far, I am not very technical and relied on 'step by step' guide in your response. 

Thank you.

**Comment by ghost** [comment]
Bummer its closed. I'm having the same issue with updating node. The supplied command is not accepted as the above user has stated.
